apps/charterafrica/app.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "charterafrica",
+  "scripts": {
+    "predeploy": "export BASE_URL=$(dokku config:get charterafrica BASE_URL) && export API_KEY=$(dokku config:get charterafrica API_KEY)",
```

Comment:
@koechkevin the payload config variables should be `PAYLOAD_SECRET_KEY` and `PAYLOAD_PUBLIC_APP_URL`